frontend/src/components/Mesh/TesterTracingConfig.tsx

```diff
@@ -0,0 +1,139 @@
+import * as React from 'react';
+import { useKialiTranslation } from '../../utils/I18nUtils';
+import { Button, ButtonVariant, Spinner, Tooltip, TooltipPosition } from '@patternfly/react-core';
+import AceEditor from 'react-ace';
+import { Theme } from '../../types/Common';
+import { istioAceEditorStyle } from '../../styles/AceEditorStyle';
+import { aceOptions, yamlDumpOptions } from '../../types/IstioConfigDetails';
+import { getKialiTheme } from '../../utils/ThemeUtils';
+import { dump, loadAll, YAMLException } from 'js-yaml';
+import ReactAce from 'react-ace/lib/ace';
+import { ValidationTypes } from 'types/IstioObjects';
+import { kialiStyle } from '../../styles/StyleUtils';
+import { Validation } from '../Validations/Validation';
+import * as API from '../../services/Api';
+
+type CheckModalProps = {
+  configData?: unknown;
+};
+
+const healthStatusStyle = kialiStyle({
+  marginLeft: '0.5rem'
+});
+
+export const TesterTracingConfig: React.FC<CheckModalProps> = (props: CheckModalProps) => {
```

Comment:
This filename has the same "Test" naming issue.  Can we change the filename and component to `CheckerTracingConfig` or something like that?